### DISCUSSION TITLE
Students unable to accept assignment in GitHub Classroom - repository access / linking error

### DISCUSSION THREAD

**Discussion by ashleykolodziej** [community/community#108432] [GitHub Education] [answered]
### Hi everyone,

Hoping for any help I can get here as quickly as possible.

My students are unable to start an assignment I've set up in Github Classroom. After they accept the assignment, they receive a 500 error. I have reproduced this behavior on my end by accepting the assignment, and then re-visiting the accept assignment link, which says I cannot access my own repository.

I am in the new assignments beta, and until this point it has been working fine. I can see that the repository fork exists from my template, but Github Classroom is not able to make the connection to the assignment repository. Any ideas?

The link to accept the assignment is: https://classroom.github.com/a/pUXw9AII

The template repository is a public template, available here, and is in my Github Education organization: https://github.com/ProfessorKolodziej/cm523-practice-2-breaking-down-css

The link to view my repository from Github Classroom is: https://classroom.github.com/publish_hydro_assignment_page_click_event?assignment_id=557156&icon_id=6&repo_id=13879858 (this will produce the 500 error)

The assignment repository I created exists here: https://github.com/ProfessorKolodziej/practice-2-breaking-down-css-ashleykolodziej

I've tried checking my education status, recreating the assignment from scratch, and using both source and template importer options. Nothing is working. Any ideas? I'm really in a bind here. Already opened a ticket with GitHub support (ticket ID: `2599757`), but I need all the help I can get right now to get this back up and running because it's blocking all my classes. These students don't know about forking yet and I really don't want to add it into the mix - we're just trying to learn basic HTML and CSS right now.

**Comment by ashleykolodziej** [comment, marked as answer]
Alright - I was able to figure out how to opt out of the beta, and things are working again.

**Reply by sfogarty** [reply]
How did you opt out of the beta? I am having similar issues.

**Comment by justin-firestone** [comment]
I am having this very same issue right now with my own GitHub Classroom. How did you opt out of the beta?
